core/chains/evm/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -305,12 +310,12 @@ func (b *BulletproofTxManager) CreateEthTransaction(newTx NewTx, qs ...pg.QOpt)
 			return err
 		}
 		err := tx.Get(&etx, `
-INSERT INTO eth_txes (from_address, to_address, encoded_payload, value, gas_limit, state, created_at, meta, subject, evm_chain_id, min_confirmations, pipeline_task_run_id, simulate)
+INSERT INTO eth_txes (from_address, to_address, encoded_payload, value, gas_limit, state, created_at, meta, subject, evm_chain_id, min_confirmations, pipeline_task_run_id, transmit_checker)
```

Comment:
Ditto here on line length.

Reply:
I don't think this is terrible. Most people can turn on wrap mode in their editor if they really must.